qemu/tests/cfg/hotplug_unplug_during_io_repeat.cfg

```diff
@@ -14,7 +14,7 @@
     iozone_timeout = 7200
     sleep_time = 30
     repeat_time = 100
-    q35:
+    q35, arm64-pci, arm64-mmio:
```

Comment:
mmio should not use PCIE bus, I think can move it out.

Reply:
Thanks for the reminder, corrected.